ui/packages/catalog-ui-search/src/main/webapp/react-component/location/bounding-box.js

```diff
@@ -150,22 +154,24 @@ const BoundingBoxLatLonDms = props => {
     const label =
       key.includes('East') || key.includes('West') ? 'dmsLon' : 'dmsLat'
     const { error, message, defaultValue } = validateGeo(label, value)
-    if (type === 'blur') {
-      setDmsError({
-        error: value !== undefined && value.length === 0,
-        message,
-        defaultValue,
-      })
-    } else if (defaultValue) {
-      setDmsError({
-        error,
-        message,
-        defaultValue,
-      })
+    setDmsError({
```

Comment:
are we preserving the old logic where the "coordinates cannot be empty" error will only show up after clicking out?

Reply:
Good catch, I just pushed a commit that brought that back